components/resources/brave_components_resources.grd

```diff
@@ -75,6 +75,7 @@
       <part file="brave_flags_ui_resources.grdp" />
       <part file="ipfs_resources.grdp" />
       <part file="domain_block_resources.grdp" />
+      <part file="request_otr_resources.grdp" />
```

Comment:
shouldn't this also have a buildflag guard?

Reply:
added